sentry-logback/src/main/java/io/sentry/logback/SentryAppender.java

```diff
@@ -38,6 +40,12 @@
      * Name of the {@link Event#extra} property containing the Thread name.
      */
     public static final String THREAD_NAME = "Sentry-Threadname";
+
+    /**
+     * Appender encoder.
+     */
+    protected Encoder<ILoggingEvent> encoder;
```

Comment:
Shouldn't this be at least `AtomicReference` and CAS'd on the use sites so that the unsynchronized access is somewhat handled? Or is it only ever supposed to be mutated by logback during instantiation and before any logging can occur?

Reply:
as far as I understand it this isn't an issue. 
logstashencoder is doing the same thing: 
https://github.com/logstash/logstash-logback-encoder/blob/bbd8333120f274f4d9ef12e797eae2331fed1a52/src/main/java/net/logstash/logback/appender/AbstractLogstashTcpSocketAppender.java#L199